mazda/installer/tweaks.sh

```diff
@@ -142,7 +142,10 @@ modify_cmu_files()
                 log_message "backup failed so leaving file as is - there will be no autostart. FAILED\n"
             fi
         else
-            log_message "backup exists so leaving file as is - there will be no autostart. FAILED\n"
+            log_message "backup exists ... "
+            echo "# Android Auto start" >> /jci/scripts/stage_wifi.sh
```

Comment:
This is a little sketchy since we aren't sure what other modifications are done to the file, but maybe it might help people who are having problems with it not auto-starting to having their files in a weird state. I dunno, but seems reasonable I guess.